src/daft-core/src/array/ops/as_arrow.rs

```diff
@@ -15,7 +15,10 @@ use crate::{
 pub trait AsArrow {
     type Output;
 
-    // Retrieve the underlying concrete Arrow2 array.
+    /// Retrieve the underlying internal Arrow2 array.
```

Comment:
```suggestion
    /// Retrieve the underlying internal Arrow2 array.
    
```
clippy likes one new line extra for short desc